oarepo_doi/services/provider.py

```diff
@@ -65,6 +66,19 @@ def url(self):
     def specified_doi(self):
         return current_app.config.get("DATACITE_SPECIFIED_ID")
 
+    @property
+    def get_serializer(self):
+        entrypoint_response_handlers = {}
+        for x in importlib_metadata.entry_points(
+                group="invenio.documents.response_handlers"
```

Comment:
This should not be hardcoded to documents, and should work with multiple models. Please see RDM_MODELS and take the response handlers from there

Reply:
Still here?